cli/module_loader.rs

```diff
@@ -221,13 +235,22 @@ impl ModuleLoadPreparer {
   }
 }
 
+#[derive(Clone)]
 struct PreparedModuleLoader {
   emitter: Arc<Emitter>,
   graph_container: Arc<ModuleGraphContainer>,
   parsed_source_cache: Arc<ParsedSourceCache>,
 }
 
 impl PreparedModuleLoader {
+  pub fn for_worker(&self, graph_container: Arc<ModuleGraphContainer>) -> Self {
```

Comment:
@littledivy I'm going to do a commit to this PR because I was just looking at this code for making more stuff an `Rc`.